kebechet/managers/version/version.py

```diff
@@ -226,8 +226,29 @@ def run(  # type: ignore
                         ["git", "push", "origin", f"refs/tags/{tag_version}"]
                     )
                 )
-            return
+                try:
+                    # Branch name is same as tag version
+                    branch_to_delete = tag_version
+                    _LOGGER.info(f"Deleting branch {branch_to_delete}")
+                    repo.git.execute(
+                        [
+                            "git",
+                            "push",
+                            "origin",
+                            "-d",
+                            f"refs/heads/{branch_to_delete}",
+                        ]
+                    )
+
+                except Exception:
+                    _LOGGER.exception(f"Failed to delete branch {branch_to_delete}")
 
+                    self.project.create_issue(
+                        title="Failed to cleanup the release version branch",
+                        body="Failed to delete branch by kebechet bot after tag creation",
+                        labels=self.labels,
+                    )
```

Comment:
As we are in the PR, let us just comment on the PR, that the bot cant delete the branch due to lack of permissions.

Reply:
yes the user would be able to see this info.
comments are allowed on closed PRs.

Reply:
It sends a notification to all the participants of the PR.(reviewers, approvers, repo maintainers, subscribers).
The issue in my opinion would need more handle like, if we open an issue in one repo, then we check if there is an existing one and the same kinda scenario.
also if the user doesn't want to give permission then this way there would only be comments. In case of an issue, we don't have a feedback loop.